src/connector/sqlite.rs

```diff
@@ -254,6 +254,26 @@ mod tests {
         assert!(res.is_empty());
     }
 
+    #[tokio::test]
```

Comment:
This is great!  Thank you for adding this.  In your the `quaint::ast::Select::value` documentation you use the following example:

```sql
    SELECT 
		`cats`.*,
		(`dogs`.`age` - ?)
	FROM `cats`
	INNER JOIN `dogs`
		ON `dogs`.`slave_id` = `cats`.`master_id`",
```

Would it be possible to use this to alias the `(dogs.age - ?)` expression?

Reply:
Yeah if it's an expression it holds the `Aliasable` trait now.
```rust
value((col!("dogs", "age") - val!(4)).alias("foo"))
```
Btw. I don't really like using these macros, but I also think I'm really reaching the edges how much I can bend the Rust type system here... :D